styles/app.css

```diff
@@ -0,0 +1,62 @@
+* {
+box-sizing: border-box;
+}
+
+*::before, *::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+}
+.container {
+width: 100%;
+border: 1px lightgrey solid;
+}
+.row {
+
+}
+.column {
+  background: white;
+  color: black;
+  padding: .1rem;
+  width: 33%;
+  display: inline-block;
```

Comment:
For this layout use `display: block` (which is the default for `div` elements) along with floats and margin for white spacing.

`display: inline-block` can give fast wins to make a column layout, but it is fairly unpredictable since the space between elements depends entirely on the font-size, font-family, etc.